src/main/java/food/finder/foodfinder/user/UserController.java

```diff
@@ -18,6 +20,11 @@
     @PostMapping("/createNewUser")
     Mono<User> createNewUser(@RequestBody User user) {
        return userService.makeSureNewUserDoesNotAlreadyExist(user.getEmail())
-               .then(userRepository.save(user));
+               .then(saveUser(user));
+    }
+
+    Mono<User> saveUser(User user) {
```

Comment:
I'm going ot have to do more research on Mono, quick google got me a lil mind boggled